src/api/data/cache/cache.go

```diff
@@ -127,18 +127,18 @@ func (c *cachedCharts) Search(params operations.SearchChartsParams) ([]*models.C
 // Refresh is the interface implementation for data.Charts
 // It refreshes cached data for all authoritative repository+chart data
 func (c *cachedCharts) Refresh() error {
-	c.rwm.Lock()
```

Comment:
Are you sure we want to remove mutex locking here? Do we want to enable concurrent `Refresh()` executions?